summarization/web_entity_detection.py

```diff
@@ -106,11 +155,17 @@ def _format_single_request(self, url: str) -> dict:
 
     def _get_best_guess_label(self, image_response):
         ''' return the best guess label '''
+        if 'error' in image_response or "label" \
```

Comment:
It is generally a good practice to use the same type of quotes (' ' or " ") throughout the code.

Reply:
Updated accordingly!